_data/accessibility-tests/text-input.yml

```diff
@@ -0,0 +1,80 @@
+      Text input instructions don't rely on visual cues, such as "in the area below." 
+      If visual instructions exist, they are announced through an ARIA label.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.3
+  - summary: Text inputs work on mobile devices. 
+    summary_additional: |
+      On mobile devices and tablets, text inputs should work in both portrait and 
+      landscape orientation with no loss of functionality.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.4
+  - summary: Color is not the only method used to convey meaning.
+    summary_additional: When you use text input fields, you never have to rely only on color to understand content.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+# Zoom/screen magnification tests
+  - summary: Text inputs remain functional when screen is magnified.
+    summary_additional: When you zoom to 200%, the text area remains visible and functional.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Focus indicators are clearly visible around text inputs.
+    summary_additional: When you use a keyboard to tab into a text input, you see an indicator around the input field.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: Text field is accesible via keyboard navigation.
+    summary_additional: Using only a keyboard, you can navigate to the text field, type in it, and navigate away from it.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Fields and labels are read in the same order that they appear on the page. 
+    summary_additional: |
+      When you use a screen reader and navigating through input fields, you hear each text field and 
+      label announced in the same order in which they appear on the page (e.g., left to right and 
+      top to bottom for English pages).
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.2
+  - summary: Screen reader announces all labels.
+    summary_additional: | 
+      When you use a screen reader and navigate to a text input, you hear every label 
+      or title associated with that text field.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0 
+    wcag_criterion: 2.4.6
+  - summary: Screen reader announces field labels and instructions.
+    summary_additional: |
+      When you use a screen reader and navigate to a text input, you hear the label or 
+      instructions that appear above the text area. (For example, "Enter first name.")
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
```

Comment:
Ehh...they do seem sound similar. However, I think they would need to be separated still. The first one is related to like general titles/headings. Example: Personal information. Where as the second is related to the specifics of each text input. Example: Enter first name. Maybe there is something we can add in there that would show how general the first one is?

Reply:
Looking deeper into this, it looks like both `2.4.6` and `3.3.2` are more related to the presentation of information than the screen reader interaction. Both criteria mention `1.3.1` as the criterion to make sure labels are appropriately marked up and related to each input field.

> This Success Criterion does not require that labels or instructions be correctly marked up, identified, or associated with their respective controls - this aspect is covered separately by [1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships). It is possible for content to pass this Success Criterion (providing relevant labels and instructions) while failing Success Criterion 1.3.1 (if the labels or instructions aren't correctly marked up, identified, or associated).

I honestly see a solution where we remove `2.4.6` from the component page since this criterion seems to be related to the general layout of pages and less the component itself. `3.3.2` could be moved as a conditional general test since it requires users to provide instructions, _but not_ correctly link them to the input.

We'd then add `1.3.1` as a screenreader test that requires screenreaders to read labels and related information correctly.

Alternatively, happy to include all three tests to cover all grounds!

Reply:
As discussed in an internal meeting, I removed `2.4.6` in favor of `3.3.2`

Resolved in [6834c19](https://github.com/uswds/uswds-site/pull/2697/commits/6834c19b70ffa20d593c9f1ddfd654317645b07f)